README.md

```diff
@@ -133,12 +133,9 @@ it through `sha256sum` with a command such as:
 $ grep node-vx.y.z.tar.gz SHASUMS256.txt | sha256sum -c -
 ```
 
-_(Where "node-vx.y.z.tar.gz" is the name of the file you have
-downloaded)_
-
-Additionally, Current and LTS releases (not Nightlies) have the GPG
-detached signature of SHASUMS256.txt available as SHASUMS256.txt.sig.
-You can use `gpg` to verify that SHASUMS256.txt has not been tampered with.
+Current and LTS releases (but not Nightlies) also have the GPG detached
```

Comment:
`excluding Nightlies` sounds more correct than `but not Nightlies`

Reply:
"but not Nightlies" sounds better to me because the two preceding items do not include (or in any way reference) Nightlies so "excluding" doesn't make sense.